design.md

```diff
@@ -1,13 +1,41 @@
 # Design
 
+asciibot
+
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+Not one. We hope no one ever uses this. This is a example of what not to do.
+
+Potentially a very visual person would want it. It can be nice to read,
+although it sucks to write.
```

Comment:
It doesn't feel any nicer than regular Picobot to read to me, but conceivably I'm not a visual enough person to be the target audience